nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -2919,4 +2919,31 @@ public void supportEnsuresAndRequiresNonNullContract() {
             "}")
         .doTest();
   }
+
+  @Test
+  public void overridingNativeModelsInAnnotatedCodeDoesNotPropagateTheModel() {
+    // See https://github.com/uber/NullAway/issues/445
+    compilationHelper
+        .addSourceLines(
+            "NonNullGetMessage.java",
+            "package com.uber;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class NonNullGetMessage extends RuntimeException {",
+            "  NonNullGetMessage(final String message) {",
+            "     super(message);",
+            "  }",
+            "  @Override",
```

Comment:
Do we want to write a test that we report an error when there is an override of `getMessage()` with no annotations and it returns `null`?

Reply:
Done.